### PR TITLE
[LogicApp] Cleanup Logic App tests

### PR DESCRIPTION
**Pull request by refortie**
## Description

Bringing all the tests in the Logic App area in line with each other. That includes renaming some of the tests, renaming some of the resources, removing hard coded location logic (#2539), and removing a bunch of extra whitespace.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by refortie** [comment]
There are no changes in this that should be visible to end users, is there a need to add anything to the changelog?

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1 [hunk 1]
```diff
@@ -300,76 +288,77 @@ function Test-ValidateLogicApp
...
 
 	# Test 1: Using physical file.
-	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $locationName -DefinitionFilePath $definitionFilePath -ParameterFilePath $parameterFilePath
+	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $location -DefinitionFilePath $definitionFilePath -ParameterFilePath $parameterFilePath
 	
 	# Test 2: Using definition object and parameter file.
 	$definition = [IO.File]::ReadAllText($definitionFilePath)
-	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $locationName -Definition $definition -ParameterFilePath $parameterFilePath
-
-	# Test 3: Failure for an invalid definition.
```
Comment (maddieclayton) [inline src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1]: Why are you removing this test?

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1 [hunk 2]
```diff
@@ -184,23 +188,7 @@ function Test-CreateAndGetLogicAppUsingDefinitionWithActions
 	$workflow3 = Get-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -Version $workflow1.Version
 	Assert-NotNull $workflow3
 
-	# Test 4: Get all workflows in ResourceGroup
```
Comment (maddieclayton) [inline src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1]: Why are you removing these three tests?

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1 [hunk 3]
```diff
@@ -300,76 +288,77 @@ function Test-ValidateLogicApp
...
 {
 	$resourceGroup = TestSetup-CreateResourceGroup
 	$workflowName = getAssetname
 	$resourceGroupName = $resourceGroup.ResourceGroupName
+	$location = Get-Location "Microsoft.Logic" "workflows" "West US"
+
 	$definitionFilePath = Join-Path $TestOutputRoot "Resources" "TestSimpleWorkflowDefinition.json"
 	$parameterFilePath = Join-Path $TestOutputRoot "Resources" "TestSimpleWorkflowParameter.json"
-	$Plan = "B9F87338CAE4470F9116F3D685365748"
-	try
```
Comment (maddieclayton) [inline src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1]: Why is this removed?

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1 (hunk 2 above)
Comment (refortie) [reply src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1]: That was a copy paste mistake. Will add back in

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1 (hunk 1 above)
Comment (refortie) [reply src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1]: Copy and paste mistake. I had forgot I had changed this in the last PR. Will fix

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1 (hunk 3 above)
Comment (refortie) [reply src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1]: This is more of the dud App Service plan code. This try catch doesn't even actually use the App Service plan in the New-AzLogicApp, so the catch would never fire
